platform/sky/contiki-sky-main.c

```diff
@@ -189,65 +189,6 @@ set_gateway(void)
 }
 #endif /* NETSTACK_CONF_WITH_IPV4 */
 /*---------------------------------------------------------------------------*/
-static void
```

Comment:
There are some modifications of contiki-sky-main.c that might be good to mention in the PR description as well.